models/User.js

```diff
@@ -0,0 +1,77 @@
+const mongoose = require('mongoose');
+const validator = require('validator');
+const bcrypt = require('bcryptjs');
+
+const UserSchema = new mongoose.Schema({
+	firstName: {
+		type: String,
+		required: [true, 'Please enter your first name'],
+		minlength: 3,
+		maxlength: 30,
+	},
+	lastName: {
+		type: String,
+		required: [true, 'Please enter your last name'],
+		minlength: 3,
+		maxlength: 100,
+	},
+	email: {
+		type: String,
```

Comment:
It should be checked with a Regex. See: https://stackoverflow.com/questions/18022365/mongoose-validate-email-syntax

Reply:
Here I used validator package. See: https://www.npmjs.com/package/validator